api/controllers/TagController.js

```diff
@@ -58,9 +58,10 @@ module.exports = {
     .then(ct => {
       if (!ct) return res.notFound()
       return res.ok(merge(
-        ct.pick('id', 'description', 'community_id'),
+        ct.pick('description', 'community_id'),
         presentWithPost(tag),
         {
+          id: ct.get('tag_id'),
```

Comment:
@levity - We're not currently using id, in tagsByCommunity, the tags are keyed by name. I was just returning id here out of habit, and maybe one day we'll need it.

Maybe I should just take id out of here for now?